assets/src/components/Line.js

```diff
@@ -7,19 +7,12 @@ function Line (props) {
     left,
     color,
     width = 2,
-    labelPlacement
+    labelPlacement // Value should be one of 'up1', 'down1', and 'down2'.  1 or 2 represents an aproximation of lines of text of offset up or down.
```

Comment:
`labelPlacement // Value should be one of 'up1', 'down1', and 'down2'.  1 or 2 represents an approximation of lines of text of offset up or down.`